### ISSUE TITLE
Updating to expo sdk 33.  build is successful but Error with googlesignin on app start 

### ISSUE THREAD

**Issue by shubhamdeol** [expo/expo#4504]
I followed all steps of updating to **expo-sdk 33** from **32**

firstly I was getting build error about can not resolve module expo-react-native-adapter , so I did 
**yarn add expo-react-native-adapter**

now I my build is successful but getting this error on app start.

```
undefined is not an object (evaluating '_ExpoGooogleSignin.default.Errors')
```

**Comment by tsapeta** [comment]
Hey,
Firstly, you shouldn't have to install `expo-react-native-adapter`. It's the old name, now we call it `@unimodules/react-native-adapter`.
In the error you posted, there are three `o` in `Google` 😄 Isn't a typo in your code?
Also, how do you import `expo-google-sign-in`?

**Comment by shubhamdeol** [comment]
Hi, that was a typo in the issue I created, not in my code, sorry about that :)
expo install expo-google-sign-in, runs the app successfully. 
but still I am not able to successfully login with google.
I have followed [this article on medium by Evan bacon](https://blog.expo.io/react-native-google-sign-in-with-expo-d1707579a7ce)  to install and use google signin.


It used to work in sdk 32, Will debug from my end and raise another issue if needed.
Thanks.
